docs/extend/modular-resources/create/_index.md

```diff
@@ -478,6 +476,20 @@ For example:
 #!/bin/sh
 cd `dirname $0`
 
+# Be sure to use `exec` so that termination signals reach the python process,
+# or handle forwarding termination signals manually
+exec $PYTHON src/main.py $@
```

Comment:
I wanted to split up the examples so that there would be one that would block less on requirements/env setup I think! If you don't think that's a good idea, happy to remove

Reply:
I don't think it'd be a good idea, if the user has multiple python modules it's going to be really hard to keep track of requirements and make sure the versions are right/don't conflict.

Reply:
Ok sounds good! Removed the less complex version!